### PR TITLE
Refactor the filenames of SOT configs

### PR DESCRIPTION
**Pull request by JingweiZhang12**

### CODE REVIEW

File: .dev_scripts/benchmark/batch_train_list.txt [hunk 1]
```diff
@@ -10,4 +10,4 @@ configs/mot/tracktor/tracktor_faster-rcnn_r50_fpn_4e_mot17-private-half.py
 configs/mot/bytetrack/bytetrack_yolox_x_crowdhuman_mot17-private-half.py
 
 # SOT
-configs/sot/siamese_rpn/siamese_rpn_r50_1x_lasot.py
+configs/sot/siamese_rpn/siamese_rpn_r50_lasot.py
```
Comment (GT9505) [inline .dev_scripts/benchmark/batch_train_list.txt:13]: Change it to `siamese_rpn_r50_20e_lasot`
We need to add the number of training epochs in the name of config

File: configs/sot/siamese_rpn/README.md [hunk 2]
```diff
@@ -36,8 +36,8 @@ We provide the best model with its configuration and training log.
...
-| SiamRPN++ <br> (FP16) |    R-50    |  -  |   20e    | -        | -              | 49.1 | 57.0 | -  | [config](siamese_rpn_r50_fp16_1x_lasot.py) | [model](https://download.openmmlab.com/mmtracking/fp16/siamese_rpn_r50_fp16_1x_lasot_20210731_110245-6733c67e.pth) &#124; [log](https://download.openmmlab.com/mmtracking/fp16/siamese_rpn_r50_fp16_1x_lasot_20210731_110245.log.json) |
+| SiamRPN++ |    R-50    |  -  |   20e    | 7.54        | 50.0              | 50.1 | 59.1 |  48.7 | [config](siamese_rpn_r50_lasot.py) | [model](https://download.openmmlab.com/mmtracking/sot/siamese_rpn/siamese_rpn_r50_1x_lasot/siamese_rpn_r50_1x_lasot_20211203_151612-da4b3c66.pth) &#124; [log](https://download.openmmlab.com/mmtracking/sot/siamese_rpn/siamese_rpn_r50_1x_lasot/siamese_rpn_r50_1x_lasot_20211203_151612.log.json) |
+| SiamRPN++ <br> (FP16) |    R-50    |  -  |   20e    | -        | -              | 49.1 | 57.0 | -  | [config](siamese_rpn_r50_fp16_lasot.py) | [model](https://download.openmmlab.com/mmtracking/fp16/siamese_rpn_r50_fp16_1x_lasot_20210731_110245-6733c67e.pth) &#124; [log](https://download.openmmlab.com/mmtracking/fp16/siamese_rpn_r50_fp16_1x_lasot_20210731_110245.log.json) |
```
Comment (GT9505) [inline configs/sot/siamese_rpn/README.md:40]: remeber to modify the fp16 config

File: configs/sot/siamese_rpn/README.md [hunk 3]
```diff
@@ -54,15 +54,15 @@ The results below are achieved without hyperparameters search.
 
 | Method |    Backbone     |  Style  | Lr schd | Mem (GB) | Inf time (fps) | Success | Norm Precision | Precision | Config | Download |
 | :-------------: | :-------------: | :-----: | :-----: | :------: | :------------: | :----: | :----: | :------: | :------: | :--------: |
-| SiamRPN++ |    R-50    |  -  |   20e    | 7.54     | -             | 59.8 | 77.3 | 80 | [config](siamese_rpn_r50_1x_uav123.py) | [model](https://download.openmmlab.com/mmtracking/sot/siamese_rpn/siamese_rpn_r50_1x_uav123/siamese_rpn_r50_1x_uav123_20211203_153949-6d82f39e.pth) &#124; [log](https://download.openmmlab.com/mmtracking/sot/siamese_rpn/siamese_rpn_r50_1x_uav123/siamese_rpn_r50_1x_uav123_20211203_153949.log.json) |
+| SiamRPN++ |    R-50    |  -  |   20e    | 7.54     | -             | 59.8 | 77.3 | 80 | [config](siamese_rpn_r50_uav123.py) | [model](https://download.openmmlab.com/mmtracking/sot/siamese_rpn/siamese_rpn_r50_1x_uav123/siamese_rpn_r50_1x_uav123_20211203_153949-6d82f39e.pth) &#124; [log](https://download.openmmlab.com/mmtracking/sot/siamese_rpn/siamese_rpn_r50_1x_uav123/siamese_rpn_r50_1x_uav123_20211203_153949.log.json) |
```
Comment (GT9505) [inline configs/sot/siamese_rpn/README.md:57]: remerber to modify configs of other datasets

File: docs/en/quick_run.md [hunk 4]
```diff
@@ -131,9 +131,9 @@ Assume that you have already downloaded the checkpoints to the directory `checkp
 
 ```shell
 python ./demo/demo_sot.py \
-    ./configs/sot/siamese_rpn/siamese_rpn_r50_1x_lasot.py \
+    ./configs/sot/siamese_rpn/siamese_rpn_r50_lasot.py \
     --input ${VIDEO_FILE} \
-    --checkpoint checkpoints/siamese_rpn_r50_1x_lasot_20201218_051019-3c522eff.pth \
+    --checkpoint checkpoints/siamese_rpn_r50_1x_lasot_20211203_151612-da4b3c66.pth \
```
Comment (GT9505) [inline docs/en/quick_run.md:136]: the name still contains  `_1x_`

File: configs/sot/siamese_rpn/README.md (hunk 2 above)
Comment (JingweiZhang12) [reply configs/sot/siamese_rpn/README.md:40]: all the file names that contain '1x' are model links or log links. It doesn't need to be changed?

**Review by GT9505 — APPROVED** [review]

**Pull request merged**